src/dropdown.option.test.events.ts

```diff
@@ -53,15 +54,18 @@ it('dispatches a "private-value-change" event', async () => {
 it('does not allow its "toggle" event to propagate', async () => {
```

Comment:
This test was falsely passing before. Fixed that and made it slightly more realistic.